Aufgabenteil Jonas Ka. (#5)
fsphys-muenster/ersti-fibel: tex/artikel/Mensa.tex

Alltäglich gegen Mittag beginnt der Magen zu knurren.
Für die Leute, die nicht jeden Tag nach Hause fahren oder selber kochen können -- also wenn Mutti zu weit weg wohnt -- gibt es die Mensa.
Das Gute ist, dass die Mensa~am~Ring) direkt nebenan liegt.
Die meisten Studierenden gehen zwischen 11:45~Uhr und 12~Uhr direkt nach den Vorlesungen in die Mensa.

> Dort erwartet euch dann meist eine große Schlange, denn um 12~Uhr ist der Anlauf dort am größten, denn die Chemiker machen Vorlesung bis Punkt zwölf und kombinieren sich mit Leuten, die von weiter weg laufen und Trödlern. Ältere Semester (oder Leute die dies hier gelesen haben) werden sich daher entweder gegen 11:45~Uhr beeilen, oder wenn möglich ein wenig warten. Um etwa 12:20~Uhr ist es meist schon wieder wesentlich leerer.

Reviewer: Du hast die stilistisch schönere Variante gefunden die nein HIrn blockiert hat

Doch diese Schlange ist meist nach kurzer Zeit überwunden und da man normalerweise in Gruppen essen geht auch nicht so schlimm. Schwieriger ist da die Platzsuche, insbesondere für größere Gruppen. Gegen 12~Uhr ist die Mensa am vollsten, sodass vier Personen probleme kriegen können zusammen zu sitzen, danach lehrt sie sich langsam wieder. um etwa 12:20~Uhr wird man kaum noch Sitzplatzprobleme haben.
Bevor man sich anstellt, sollte man allerdings so einiges über die Mensa wissen!

Seit dem Sommersemester 2017 benötigt man zum Bezahlen lediglich seinen Studierendenausweis.
Das Guthaben darauf kann an den Automaten im Foyer der Mensa aufgeladen werden.
Wenn das Guthaben nicht mehr ausreicht um euer Essen zu bezahlen, könnt ihr den Ausweis auch an der Kasse noch aufladen. 
Das ist jedoch mit einem kleinen Aufpreis behaftet und die Missgunst der Kassiererin, sowie aller die in der Schlange hinter euch stehen ist euch sicher.
Falls ihr den Studierendenausweis noch nicht zugeschickt bekommen habt, könnte das daran liegen, dass ihr noch kein Foto dafür hochgeladen habt.

> Das solltet ihr dann so bald wie möglich nachholen (textit{insbesondere weil ihr diesen auch als Bibliotheksausweis und für die Benutzung eures Semestertickets braucht}.

Reviewer: not shure if necessary, aber mein sinn für kommasetzung ist nicht mehr, was er mal war

> Das solltet ihr dann so bald wie möglich nachholen (textit{insbesondere weil ihr diesen auch als Bibliotheksausweis und für die Benutzung eures Semestertickets braucht}.

Author: Okay, kann ich drauf verzichten, definitiv muss da aber eine schließende Klammer rein:
```suggestion
Das solltet ihr dann so bald wie möglich nachholen (textit{insbesondere weil ihr diesen auch als Bibliotheksausweis und für die Benutzung eures Semestertickets braucht}).
```

> Das solltet ihr dann so bald wie möglich nachholen (textit{insbesondere weil ihr diesen auch als Bibliotheksausweis und für die Benutzung eures Semestertickets braucht}.

Reviewer: ok

\end{center}

einsehen oder euch wöchentlich als Newsletter per E-Mail zuschicken lassen.
Es gibt auch die Mensa-App (für Android-Geräte im Google Play Store, für iPhones im App-Store). Normalerweise geht man aber einfach los und schaut auf die
großen LCD-Bildschirme im Foyer.Dort  könnt ihr alle Tagesangebote (z.\,B.\ auch die Beilagen und Desserts) sowohl "Oben" als auch im Buffetsaal "Unten" und deren Preise erfahren. Zusatzstoffe und Allergene werden unter den Gerichten mit Zahlen bzw Buchstaben gekennzeichnet und andere eventuell wichtige Eigenschaften des Gerichts (z.\,B. "mit Schwein", "mit Fisch", "vegetarisch") werden mit Piktogrammen angezeigt. Die dazugehörige Legende hängt an vershiedenen Stellen bei Essensausgabe und Kassen.

> "Oben" werden immer 4~Menüs angeboten. Davon ist immer mindestens eine vegetarisches und ein veganes, welches an einer eigenen Theke mit eventuell eigenen Beilagen und einem kleinen Salatbuffet angeboten wird.

Reviewer: s.o.

> "Oben" werden immer 4~Menüs angeboten. Davon ist immer mindestens eine vegetarisches und ein veganes, welches an einer eigenen Theke mit eventuell eigenen Beilagen und einem kleinen Salatbuffet angeboten wird.

Reviewer: re salatbuffet: ich glaube ja, aber vieleicht habe ich das auch geträumt

Preislich liegen die Menüs bei \SIlist{2,30; 2,95; 3,30}{\euro} und beinhalten drei mehr oder minder passende Beilagen.
Jede weitere Beilage liegt bei \SI{0,25}{\euro}; lasst ihr eine Beilage weg, bedeutet das einen Nachlass von ebenfalls \SI{0,25}{\euro}.
Solltet ihr jedoch kein Menü haben, so liegt die Beilage bei \SI{0,65}{\euro}.
Abhilfe schaffen hier meistens die Bezahlung durch einen Kommilitonen mit Menü.
Die Glasschälchen mit ausgefalleneren Desserts sind für \SI{0,95}{\euro} zu haben.

Die häufigen Mensabesucher werden jedoch schnell bemerken, dass die Menüauswahl sich alle 4--6~Wochen wiederholt.

> Das ist aber gar nicht so schlimm, denn die Menüs schmecken dann wieder genauso gut oder schlecht wie zuvor; so bildet sich langsam ein kompetenter Mensabesucher heraus.

Reviewer: ich vermeide gerne direkte anrede, wenn der geneigte lektor dieser ansicht ist...

> Das ist aber gar nicht so schlimm, denn die Menüs schmecken dann wieder genauso gut oder schlecht wie zuvor; so bildet sich langsam ein kompetenter Mensabesucher heraus.

Author: ```suggestion
Das ist aber gar nicht so schlimm, denn die Menüs schmecken dann wieder genauso gut oder schlecht wie zuvor; so wird man langsam ein kompetenter Mensabesucher.
```

So besser?

> Das ist aber gar nicht so schlimm, denn die Menüs schmecken dann wieder genauso gut oder schlecht wie zuvor; so bildet sich langsam ein kompetenter Mensabesucher heraus.

Reviewer: ist sowieso ne reine geschmackssache, bin ich mit zufrieden

> Bei den ersten Besuchen solltet ihr aufpassen, dass die Bilder auf den Bildschirmen (soweit vorhanden) oftmals fast, aber eben nicht ganz nichts mit dem Aussehen des Essens zu tun haben.

Reviewer: oh well, heisst nicht umsonst 'kill your darlings', hätte ich gerne drin und ist meiner ansicht nach nicht sonderlich störend, würde aber nicht unbedingt für kämpfen

> Bei den ersten Besuchen solltet ihr aufpassen, dass die Bilder auf den Bildschirmen (soweit vorhanden) oftmals fast, aber eben nicht ganz nichts mit dem Aussehen des Essens zu tun haben.

Author: Okay, dann lassen wir den drin!

"Unten" gibt es ein vielfältiges Angebot von immer wiederkehrenden Menüs.
Jeden Tag könnt ihr an der Grillstation Currywurst mit Pommes oder ein Jäger-/Zigeunerschnitzel erwerben.

> Montags gibt es den beliebten Mensaburger (auch vegetarisch) in verschiedenen Variationen, welcher so groß ist, dass sich an ihm die Geister scheiden, ob er mit Messer und Gabel oder mit der Hand gegessen werden muss (edit: der Mensaburger wird natürlich mit Besteck gegessen, alles andere ist einfach falsch).

Author: ```suggestion
Montags gibt es den beliebten Mensaburger (auch vegetarisch) in verschiedenen Variationen, welcher so groß ist, dass sich an ihm die Geister scheiden, ob er mit Messer und Gabel oder mit der Hand gegessen werden muss. \textit{\[Anm. des Autors: Den Mensaburger ohne Besteck zu essen, ist natürlich einfach falsch!\]}
```

Vielleicht macht das hier das Ganze ein bisschen einfacher zu lesen?

Reviewer: jo

> An der Wokstation gibt es häufig Aktionen, meist aber Gerichte, die in großen Pfannen zubereitet werden können,  darunter vermutlich am wichtigsten jeden Donnerstag das allseits beliebte (wirklich, die Schlange ist ein Alptraum) Gyros mit Zaziki, viel Krautsalat und Pommes, zu dem durchaus auch Vegetarier greifen, dann aber ohne den Gyrosanteil und mit mehr Kraut.

Author: ```suggestion
An der Wokstation gibt es häufig Aktionen – meist aber Gerichte, die in großen Pfannen zubereitet werden können. Darunter vermutlich am wichtigsten ist jeden Donnerstag das allseits beliebte (wirklich, die Schlange ist ein Albtraum) Gyros mit Zaziki, viel Krautsalat und Pommes, zu dem durchaus auch Vegetarier greifen, dann aber ohne den Gyrosanteil und mit mehr Kraut.
```

Author: Tatsächlich geht `Alptraum` und `Albtraum`. Wieder was gelernt!

> Dazu gibt es eine Pastatheke, ein Buffet mit Nudeln, Auflaufen und aufgewärmten Resten (häufig eine Anlaufstelle wenn man sonst nichts mag) und ein Salatbuffet, an dem man sich auch Wraps zusammenstellen kann und das sich insbesondere im Sommer großer Beliebtheit erfreut. Allerdings eine Warnung: Die Teller vom Buffet  werden an der Kasse nach Gewicht abgerechnet (Das Ganze kostet dann \SI{0,57}{\euro} pro \SI{100}{\g}) und das wird sehr schnell mehr als man beim Zusammenstellen denkt. Außerdem ist ein überquillender Wrap sein eigenes Problem.

Author: ```suggestion
Dazu gibt es eine Pastatheke, ein Buffet mit Nudeln, Aufläufen und aufgewärmten Resten (häufig eine Anlaufstelle wenn man sonst nichts mag) und ein Salatbuffet, an dem man sich auch Wraps zusammenstellen kann und das sich insbesondere im Sommer großer Beliebtheit erfreut. Allerdings eine Warnung: Die Teller vom Buffet werden an der Kasse nach Gewicht abgerechnet (Das Ganze kostet dann \SI{0,57}{\euro} pro \SI{100}{\g}) und das wird sehr schnell mehr als man beim Zusammenstellen denkt. Außerdem ist ein überquillender Wrap sein eigenes Problem.
```

Reviewer: ?

Author: Auflaufen -> Aufläufen (a->ä)

Reviewer: ah!

Falls ihr Pommes nehmt, ist es empfohlen, mit Pommessalz nachzusalzen und gegen einen kleinen Aufpreis Majo oder Ketchup zu erstehen, Senf ist wie in einer richtigen Frittenbude kostenlos.
Des Weiteren gibt es fast immer einen sehr günstigen Eintopf an der passend genannten Eintopfstation.
Zu jedem Essen gibt es ein recht gutes Angebot an preiswerten Getränken.

> Häufig gibt es (meist an der Wokstation) verschiedene Aktionen, so etwa regelmäßig Pizza, deren tatsächlicher Belag nichts mit der Benennung zu tun hat und in der jeweiligen Saison z.\,B. Spargelgerichte oder Erdbeeren mit Schlagsahne. Seit neuerem bietet die Mensa außerdem regelmäßig Friedensteller an. Dies sind Gerichte, die nach Rezepten der Friedensteller-Intiative (überraschend, ich weiß) gekocht wurden und sich durch Nachhaltigkeit auszeichnen, z.\,B. durch Saisonalität und Regionalität der Zutaten. Diese Ersetzen normalerweise eine vegetarische (oder in einigen Fällen vegane) Auswahl "Oben".

Author: ```suggestion
Häufig gibt es (meist an der Wokstation) verschiedene Aktionen, so etwa regelmäßig Pizza, deren tatsächlicher Belag nichts mit der Benennung zu tun hat und in der jeweiligen Saison z.\,B. Spargelgerichte oder Erdbeeren mit Schlagsahne. Seit neuerem bietet die Mensa außerdem regelmäßig Friedensteller an. Dies sind Gerichte, die nach Rezepten der Friedensteller-Intiative (überraschend, ich weiß) gekocht wurden und sich durch Nachhaltigkeit auszeichnen, z.\,B. durch Saisonalität und Regionalität der Zutaten. Diese ersetzen normalerweise eine vegetarische (oder in einigen Fällen vegane) Auswahl "Oben".
```

Author: neuerem -> Neuerem und irgendwas anderes, das ich gerade nicht mehr finde.

Reviewer: ah, ok

	\includegraphics[width=\columnwidth]{res/xkcd/720_recipes.png}
}{\url{https://xkcd.com/720}}

> Falls jedoch jemand gar nichts gefunden haben sollte, kann man es auch mit einem Brötchen oder Rühr- oder Spiegelei im Café Viva versuchen.

Author: ```suggestion
Falls jedoch jemand gar nichts gefunden haben sollte, kann man es auch mit einem Brötchen oder einer Waffel im Café Viva versuchen.
```

Die hatten mal Rührei?

Reviewer: actually an oversight, schien beim rübergucken passend genug

Reviewer: anscheinend, zumindest habe ich häufiger rühreibrötchen gesehen

Hier werden auch die meisten anstehenden Sportereignisse übertragen und morgens in der Wiederholung gezeigt.
Außerdem kann man hier recht entspannt sitzen und etwa eine Kaffeespezialität trinken.

> Im Foyer sind außerdem noch ein kleiner Kiosk mit Getränken und Süßigkeiten und ein Eismann, welcher im Winter auch leckere Waffeln anbietet.

Author: ```suggestion
```

GIbt es nicht mehr.

Reviewer: Der ist endgültig weg? jenseits eigener anschauung dazu leider keine info

Author: Ja, leider offenbar.

Reviewer: insert sadface

> Dazu kommen noch diverse Läden wie ein sehr nützlicher Copyshop und Schreibwahrenladen (Kopieren und Drucken ist allerdings preiswerter, wenn man Print~\&~Pay beim ZIV nutzt oder mit dem Guthaben des Studierendenausweis an den Kopierern z.\,B.\ im Lernzentrum, der IG1 oder im ZIV. Aber vergesst eure Karte dort nicht!), eine Filiale der Techniker-Krankenkasse und der Aster-Reiseservice.

Author: ```suggestion
Dazu kommen noch diverse Läden wie ein sehr nützlicher Copyshop und Schreibwahrenladen (Kopieren und Drucken ist allerdings preiswerter, wenn man Print~\&~Pay beim ZIV nutzt oder mit dem Guthaben des Studierendenausweis an den Kopierern z.\,B.\ im Lernzentrum, der IG1 oder im ZIV. Aber vergesst eure Karte dort nicht!). Außerdem gibt es noch eine Filiale der Techniker-Krankenkasse und den Aster-Reiseservice.
```

Reviewer: jo
